misk-redis/src/main/kotlin/misk/redis/Redis.kt

```diff
@@ -439,6 +440,16 @@ interface Redis {
    * Closes the client, so it may not be used further.
    */
   fun close()
+
+  /**
+   * Subscribe to a redis channel via pubsub. This is blocking!
+   */
+  fun subscribe(jedisPubSub: JedisPubSub, channel: String)
```

Comment:
Possibly I shouldn't be bleeding this interface to the world, but I couldn't figure out a nice way to avoid this, lemme know if anyone has a better way lmao

Reply:
hot take: As someone who used these Redis APIs a lot, it was always annoying when our attempt at encapsulation got in the way of just exposing the full capabilities of the Redis API.

All to say, leaking it doesn't bother me. :)